e2e-tests/specs/relay-general-functionality.spec.ts

```diff
@@ -9,32 +9,39 @@ test.describe('Free - General Functionalities, Desktop', () => {
     await checkAuthState(page)
     await dashboardPage.maybeDeleteMasks()
   });
-  
-  test('Check the free user can only create 5 masks, C1553067', async ({ dashboardPage }) => {
+
+  test('Check the free user can only create 5 masks, C1553067', async ({ dashboardPage, page }) => {
     // Generating five masks takes a while:
     test.slow()
-    await dashboardPage.generateMask(5)
-    
+    await expect(async () => {
+      await dashboardPage.generateMask(5)
+      expect(await page.locator(dashboardPage.maskCard).count() === 5)
+    }).toPass()
+
+
     // After five times, the button becomes greyed-out and the user cannot add other masks anymore (TODO: for a free user from a country where Premium is NOT available).
     expect(await dashboardPage.maxMaskLimitButton.textContent()).toContain('Get unlimited email masks')
   })
-  
-  test('Check that when generating a new mask, its card is automatically opened, C1686210, C1553075, C1553064', async ({ dashboardPage }) => {
-    await dashboardPage.generateMask(1)
-    
-    await expect(dashboardPage.maskCardExpanded).toBeVisible()
+
+  test('Check that when generating a new mask, its card is automatically opened, C1686210, C1553075, C1553064', async ({ dashboardPage, page }) => {
+    await expect(async () => {
+      await dashboardPage.generateMask(1)
+      expect(await page.locator(dashboardPage.maskCard).count() === 1)
+    }).toPass()
+
+    await expect(dashboardPage.maskCardExpandButton).toHaveAttribute("aria-expanded", "true")
```

Comment:
Nice, `[aria-expanded]` should keep working as long as we have expand functionality :)